src/toolkits/style_transfer/style_transfer.cpp

```diff
@@ -523,7 +527,10 @@ void style_transfer::init_train(gl_sarray style, gl_sarray content,
 
   infer_derived_options();
 
-  add_or_update_state({{"model", "resnet-16"}});
+  add_or_update_state({
+    {"model", "resnet-16"},
+    {"style_sarray", style},
```

Comment:
The Python implementation stores this as an SFrame named "styles". Since we must support that representation for any already existing models, why not just adopt that here?

https://github.com/apple/turicreate/blob/25eab55fd492f355f8667ddd2cc60e26697d8681/src/python/turicreate/toolkits/style_transfer/style_transfer.py#L475